.github/workflows/simulation-1.1.yml

```diff
@@ -184,7 +184,7 @@ jobs:
     - name: Build
       run: |
         ./bootstrap
-        make -f examples/Makefile-simulation
+        CPPFLAGS=-DOPENTHREAD_CONFIG_BORDER_AGENT_UDP_PORT=49191 make -f examples/Makefile-simulation
```

Comment:
Seems it's more difficult to run local test by running `make -f examples/Makefile-simulation check`.
How do we make sure that locally running `VERBOSE=1 make -f examples/Makefile-simulation check` still works (without configuring CPPFLAGS).
Should we define `OPENTHREAD_CONFIG_BORDER_AGENT_UDP_PORT=49191` for all simulation builds?